constraint/pkg/client/drivers/local/local.go

```diff
@@ -107,20 +109,20 @@ func copyModules(modules map[string]*ast.Module, filter string) map[string]*ast.
 
 func (d *driver) checkModuleName(name string) error {
 	if name == "" {
-		return errors.Errorf("Module name cannot be empty")
+		return fmt.Errorf("Module name cannot be empty")
```

Comment:
nit here and below... lowercase first letter?

Reply:
Done. Fixed about a hundred such similar cases.